includes/core/class-wc-payments-order.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Class file for WCPay\Core\WC_Payments_Order
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core;
+
+defined( 'ABSPATH' ) || exit;
+
+use \WC_Order;
```

Comment:
The backslash is not required here, `use` addresses the global namespace.

Reply:
Fixed